app/views/devise/registrations/new.html.erb

```diff
@@ -1,43 +1,45 @@
-<h2>Sign up</h2>
+<h1 class="h4 text-gray-900 mb-4">Create an Account!</h1>
```

Comment:
Use internationalization. Verify throughout the code

Reply:
done